webcomponents/tool/src/main/frontend/packages/sakai-conversations/src/SakaiComment.js

```diff
@@ -148,8 +148,8 @@ export class SakaiComment extends SakaiElement {
             <div class="post-message">${unsafeHTML(this.comment.message)}</div>
             <div class="post-reactions-block">
               ${!this.comment.locked && (this.comment.canEdit || this.comment.canDelete) ? html `
-              ${this._renderOptionsMenu()}
-              ` : ""}
+                ${this._renderOptionsMenu()}
+              ` : nothing}
```

Comment:
spacing?

Reply:
It seems okay to me. I'm using 2 spaces for tabs.